electrum/synchronizer.py

```diff
@@ -216,6 +216,7 @@ async def _request_missing_txs(self, hist, *, allow_server_not_finding_tx=False)
                 continue  # already have complete tx
             transaction_hashes.append(tx_hash)
             self.requested_tx[tx_hash] = tx_height
+            self.adb.pending_txs_changed(True)
 
         if not transaction_hashes: return
```

Comment:
I am not quite sure why the function call is inside the loop. Would it not make more sense to call it just once, outside the loop? e.g.:
```suggestion

        if not transaction_hashes: return
        self.adb.pending_txs_changed(True)
```
(although this might depend on what use case you have in mind - see other remark comment)